.github/ISSUE_TEMPLATE/new_project.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Add new Project
```

Comment:
I don't think this is the best name to give this template, I think maybe add a new feature may be better